samples/WebApiContrib.Core.Samples/Controllers/VersionedController.cs

```diff
@@ -0,0 +1,15 @@
+using Microsoft.AspNetCore.Mvc;
+using WebApiContrib.Core.Versioning;
+using WebApiContrib.Core.Samples.Model;
+
+namespace WebApiContrib.Core.Samples.Controllers
+{
+    [Route("api/versioned")]
+    public class VersionedController : Controller
+    {
+        public IActionResult Get()
+        {
+            return new VersionedObjectResult(new PersonModel("Kristian", "Hellang", age: 27));
```

Comment:
what is the purpose of `VersionedObjectResult` over `ObjectResult`? it doesn't seem to add any functionality